docs/api/python/symbol/symbol.md

```diff
@@ -207,6 +207,7 @@ Composite multiple symbols into a new one by an operator.
 
     Symbol.broadcast_to
     Symbol.broadcast_axes
+    Symbol.broadcast_like
```

Comment:
also need to add an entry at https://github.com/ifeherva/incubator-mxnet/blob/00eeeca61c9f052f3e85d4febe43130ed5669e61/docs/api/python/symbol/symbol.md#expanding-elements-1. Same goes with ndarray.

Reply:
Done